_data/projects.yml

```diff
@@ -361,4 +361,72 @@
   tags: 
     - OpenWhisk
     - Alexa
-    - Cloudant
\ No newline at end of file
+    - Cloudant
+    
+ - title:    Crafting Community Events that Leave them Wanting S'more
+  headline: A look at what makes Offline Camp's unconconference format so impactful
+  links: 
+    Medium: https://medium.com/offline-camp/crafting-community-events-that-leave-them-wanting-smore-902974cff4d4
+  features: 
+    - offline-camp    
+    
+- title:    The Offline Camp Experience 
+  headline: Our campers reveal the secret sauce that makes Offline Camp so special
+  links: 
+    YouTube: https://www.youtube.com/watch?v=FNtpPW_7H1k
+  features: 
+    - offline-camp    
+    
+ - title:   Offline Camp California 2016
+  headline: Offline Camp takes to the ranch
+  links: 
+    YouTube:  https://www.youtube.com/watch?v=2vCmlxtMvwE&t=26s
+  features: 
+    - offline-camp    
+    
+- title:   Offline Camp Berlin 2017
+  headline: Offline Camp makes its first visit to Europe
+  links: 
+    YouTube:  https://www.youtube.com/watch?v=bsQapDA--Sw
+  features: 
+    - offline-camp    
+
+    
+- title:   Recap: Offline Camp Berlin
```

Comment:
This is invalid YAML. The value here should be in quotes as it contains a colon.